aloscene/depth.py

```diff
@@ -274,3 +283,74 @@ def as_disp(
             names=self.names,
         )
         return depth
+
+    def as_euclidean(self, camera_intrinsic: aloscene.CameraIntrinsic = None, projection="pinhole", distortion=1.0):
+        """Create a new Depth augmented tensor whose data is the euclidean depth (distance) from camera to world points.
+        To use this method, we must know intrinsic matrix of camera, projection model and distortion coefficient
+        (if exists).
+
+        Parameters
+        ----------
+        camera_intrinsic: aloscene.CameraIntrinsic
+        projection: str | pinhole
+            At this moment, only 2 projections models are supported: pinhole (f*tan(theta)) and equidistant (f*theta:
+            which is
+            used for wide range camera).
+        distortion: float | 1.0
+            Distortion coefficient for equidistant model. Only linear distortion supported
+            (sensor_angle=distortion*theta).
+
+        Returns
+        -------
+        aloscene.Depth
+        """
+        planar = self
+        if not planar.is_planar:
+            raise ExecError("Cannot transform to euclidian because this is already a euclidian depth tensor.")
```

Comment:
Up to know, on other aug tensor, if the operation is call twice, we simply return self.clone() instead of raising an error